cost-analyzer/templates/cost-analyzer-deployment-template.yaml

```diff
@@ -797,18 +784,6 @@ spec:
             - name: FEDERATED_REDIRECT_BACKUP
               value: "true"
             {{- end}}
-            {{- if .Values.federatedETL.useExistingS3Config }}
-            - name: FEDERATED_USE_EXISTING_CONFIG
-              value: "true"
-            {{- end}}
-            {{- if .Values.federatedETL.federator.enabled }}
-            - name: FEDERATED_FEDERATOR_ENABLED
-              value: "true"
-            {{- end}}
-            {{- if .Values.federatedETL.federator.useMultiClusterDB }}
-            - name: CURRENT_CLUSTER_ID_FILTER_ENABLED
-              value: "true"
-            {{- end}}
```

Comment:
@nickcurie I see you delete this config here, but you're not adding it to the federator's template? Is that intentional?